woodwork/exceptions.py

```diff
@@ -78,3 +78,29 @@ class WoodworkNotInitWarning(UserWarning):
 
 class TypeValidationError(Exception):
     pass
+
+
+class LatLongIsNotTupleError(ValueError):
```

Comment:
Do we want/need separate error classes for each of these, or should we use a single class and customize the error based on the specific failure? Also, is this distinct from the existing `TypeValidationError` or could we use that instead? If we don't use `TypeValidationError` should these be a subclass of that?

I'm not sure what the right answer is to any of those questions, just raising it for discussion.